contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-dotnet-set-send-feature-flags-to-true.mdx

```diff
@@ -11,3 +13,33 @@ posthog.Capture(
     sendFeatureFlags: true
 );
 ```
+
+## Advanced usage (v6.2.0+)
```

Comment:
Is this the right dotnet version? I see https://github.com/PostHog/posthog-dotnet/releases/tag/v1.0.6 as most recent.